### PR TITLE
AdministrativeUnit.*.All and RoleManagement.*.Directory

### PR DESCRIPTION
**Pull request by VinceSmith**
Updated administrative unit API references with new permission added to permissions reference.

### CODE REVIEW

**Comment by VinceSmith** [comment]
#sign-off

File: concepts/permissions-reference.md [hunk 1]
```diff
@@ -68,6 +68,36 @@ For more information about administrator roles, see [Assigning administrator rol
...
+| _AdministrativeUnit.Read.All_ |   Read all administrative units | Allows the app to read administrative units and administrative unit membership without a signed-in user. | Yes |
+| _AdministrativeUnit.ReadWrite.All_ |   Read and write all administrative units | Allows the app to create, read, update, and delete administrative units and manage administrative unit membership without a signed-in user. | Yes |
+
+### Remarks
+
+_AdministrativeUnit.Read.All_ and _AdministrativeUnit.ReadWrite.All_ are valid only for work or school accounts.
+
+### Example usage
+
+- _AdministrativeUnit.Read.All_: Read administrative units and administrative unit membership (`GET /beta/administrativeUnits`)
```
Comment (dkershaw10) [inline concepts/permissions-reference.md]: Read all administrative units in the organization

File: concepts/permissions-reference.md [hunk 2]
```diff
@@ -68,6 +68,36 @@ For more information about administrator roles, see [Assigning administrator rol
...
+- _AdministrativeUnit.ReadWrite.All_: Create, read, update, and delete administrative units and manage administrative unit membership  (`PATCH /beta/administrativeUnits`)
```
Comment (dkershaw10) [inline concepts/permissions-reference.md]: Update an administrative unit's displayName (`PATCH /beta/administrativeUnits/{id}`)

File: concepts/permissions-reference.md [hunk 3]
```diff
@@ -68,6 +68,36 @@ For more information about administrator roles, see [Assigning administrator rol
...
+|   Permission    |  Display String   |  Description | Admin Consent Required |
+|:-----------------------------|:-----------------------------------------|:-----------------|:-----------------|
+| _AdministrativeUnit.Read.All_ |   Read all administrative units | Allows the app to read administrative units and administrative unit membership without a signed-in user. | Yes |
+| _AdministrativeUnit.ReadWrite.All_ |   Read and write all administrative units | Allows the app to create, read, update, and delete administrative units and manage administrative unit membership without a signed-in user. | Yes |
+
+### Remarks
+
+_AdministrativeUnit.Read.All_ and _AdministrativeUnit.ReadWrite.All_ are valid only for work or school accounts.
+
+### Example usage
```
Comment (dkershaw10) [inline concepts/permissions-reference.md]: I think you should add at least one more example - adding a member.  Maybe adding scopedRoleMember too.

File: concepts/permissions-reference.md [hunk 4]
```diff
@@ -1015,6 +1045,34 @@ For more complex scenarios involving multiple permissions, see [Permission scena
 
 ---
 
+## Role Management permissions
```
Comment (dkershaw10) [inline concepts/permissions-reference.md]: Please add API topic updates for these new permissions.

**Comment by VinceSmith** [comment]
#sign-off

**Comment by VinceSmith** [comment]
Let me know if the merge conflict can be resolved or if I need to discard my change and resync the changelog file.

**Comment by Lauragra** [comment]
@VinceSmith , are you ready to merge after the validation run?

**Comment by VinceSmith** [comment]
Yes, ready to merge.

**Pull request merged**